qiskit_braket_provider/providers/adapter.py

```diff
@@ -84,12 +121,26 @@
     "iswap": lambda: [braket_gates.ISwap()],
 }
 
+_qiskit_controlled_gate_names_to_braket_gates: dict[str, Callable] = {
```

Comment:
It's a private variable, so I don't think it's a big deal; I'll change the name though. As for lambdas, it's more Pythonic here _not_ to use them because we're using the direct output of the gate functions (as opposed to putting them in lists).